prime-router/src/test/kotlin/common/UniversalPipelineTestUtils.kt

```diff
@@ -184,10 +184,17 @@ object UniversalPipelineTestUtils {
     val fhirSenderWithNoTransform = UniversalPipelineSender(
         "fhir-elr-no-transform",
         "phd",
-        MimeFormat.HL7,
+        MimeFormat.FHIR,
```

Comment:
`fhirSenderWithNoTransform` was essentially a near duplicate of `hl7SenderWithNoTransform` with no ill effect across several test suites and I'm a little confused why. This value can be flipped back to HL7 without affecting test results.

Reply:
I think at this point in the pipeline we've passed all the points where that value would be checked or impact control flow. That makes sense to me since at this point it doesn't really matter what the original format was.